cmd/apiclient/client.go

```diff
@@ -0,0 +1,259 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+const BaseURL = "http://localhost:8080/books"
+
+type Book struct {
+	Title  string `json:"title"`
+	Author string `json:"author"`
+	Year   int    `json:"data"`
+}
+
+func main() {
+	for {
+		var action int
+
+		fmt.Println("Выберите команду: ")
+		fmt.Println("0 - Выход из программы")
+		fmt.Println("1 - Получить книгу по ID")
+		fmt.Println("2 - Добавить книгу")
+		fmt.Println("3 - Обновить книгу")
+		fmt.Println("4 - Удалить книгу")
+
+		sc := bufio.NewScanner(os.Stdin)
```

Comment:
```suggestion
		s := bufio.NewScanner(os.Stdin)
```

or

```suggestion
    		scanner := bufio.NewScanner(os.Stdin)
```

might be better